tests/unit/prediction/ocr/test_image_flag.py

```diff
@@ -0,0 +1,153 @@
+import pytest
+import requests
+from openfoodfacts.ocr import OCRResult
+
+from robotoff.prediction.ocr.image_flag import flag_image
+from robotoff.types import PredictionType
+
+
+@pytest.fixture
+def face_detection_ocr_result():
+    test_data_url = "https://raw.githubusercontent.com/openfoodfacts/test-data/main/robotoff/tests/unit/ocr/face_detection.json"
+    response = requests.get(test_data_url)
+    response.raise_for_status()
+    return OCRResult.from_json(response.json())
+
+
+def test_flag_image_with_face_annotation(face_detection_ocr_result):
+    predictions = flag_image(face_detection_ocr_result)
+
+    assert predictions
+
+    face_predictions = [
+        pred
+        for pred in predictions
+        if pred.type == PredictionType.image_flag
+        and pred.data.get("type") == "face_annotation"
+    ]
+
+    assert face_predictions
+    assert len(face_predictions) == 1
```

Comment:
No need to have the first assert, as we already know it's a list, both assert are redundant